Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Domain/Model/Migrations/LocalStorageContextModelSnapshot.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+
+namespace MvvmSeed.Domain.Model.Migrations
+{
+    [DbContext(typeof(LocalStorageContext))]
+    partial class LocalStorageContextModelSnapshot : ModelSnapshot
```

Comment:
Check [this](http://www.learnentityframeworkcore.com/migrations/model-snapshot).

(I don't know that much about EF to explain it myself)